.travis.yml

```diff
@@ -24,12 +28,20 @@ compiler:
   - clang
   - gcc
 
+git:
+  depth: 5
+
 env:
+  global:
+    - secure: "VezRbHFg6kllV5WG06M3tG3aHJaC3xrMylJ6RCVbL+uz2JeralVCqV7eIk4fVb9cu83Li+weEa0AJj0wkxpIUJ+vUh5F65L6gSWSbgHP7muOSVsmnEc6KvX4n3av/ZGe4geSmsxqh2pd/2xI1h7KioGRhKeqaZIdjVgWgGJW2iQ="
```

Comment:
The first is the `SONAR_TOKEN` which is needed to access the SonarQube server. The second is the encrypted `GITHUB_TOKEN`. This is the token of a GitHub user i have created that (in principal, hopefully this works) posts found issues as comments in the PR. This user can also update the PR status if the user has commits rights on the repo, which my user obviously not has. So if you find this feature useful you maybe create an own user and change the `GITHUB_TOKEN` variable.

I'll add a comment which line is which token.